TRN-105/multidim_array/multidim.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace Multidimensional;
+class ArrayDim{
+  function ComputeMultiDimArray() {
+    $array1 = [
```

Comment:
Do one thing move the `$array1` and `$array2` from here and pass as from index.php

Reply:
Done!